src/routes/landing.js

```diff
@@ -69,7 +68,7 @@ const styles = StyleSheet.create({
 
 // This value is equal to 1/(number of FullWidthContainer components)
 // It ensures the components will be equally distributed on the screen
-const columnContainerFlex = .25;
+const columnContainerFlex = .2;
```

Comment:
If this is normal flex, it should behave the same if this value is 1.

Reply:
Yes, I played around with this number and noticed no difference in affect. I just decided to keep in consistent with the other code, but think 1 is fine almost everywhere.